lib/src/widgets/image_source_sheet.dart

```diff
@@ -65,19 +66,18 @@ class ImageSourceBottomSheet extends StatelessWidget {
       preferredCameraDevice: preferredCameraDevice,
     );
     if (null != pickedFile) {
-      if (null != onImage) {
-        final image = Image.memory(await pickedFile.readAsBytes());
-        onImage(image);
-      }
-
-      if (null != onImageSelected) {
-        // Warning:  this will not work on the web platform because pickedFile
-        // will instead point to a network resource.
-        assert(!kIsWeb);
-
-        final imageFile = File(pickedFile.path);
-        assert(null != imageFile);
-        onImageSelected(imageFile);
+      if (kIsWeb) {
```

Comment:
I removed the assertion that it should only work for mobile and made it into an if-else.